src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -113,6 +113,16 @@ source %{
         if (vlen < 4) {
           return false;
         }
+      case Op_AddVHF:
+      case Op_DivVHF:
+      case Op_MaxVHF:
+      case Op_MinVHF:
+      case Op_MulVHF:
+      case Op_SqrtVHF:
+      case Op_SubVHF:
+        return UseZvfh;
+      case Op_FmaVHF:
+        return UseZvfh && UseFMA;
```

Comment:
Maybe group with the existing two cases at L98 and L99 (Op_VectorCastHF2F / Op_VectorCastF2HF)?